Gemfile

```diff
@@ -54,5 +54,6 @@ end
 group :deployment do
   gem 'capistrano-passenger', require: false
   gem 'capistrano-rails', require: false
+  gem 'capistrano-rvm' # for Ubuntu deployments
```

Comment:
```suggestion
  gem 'capistrano-rvm', require: false # for Ubuntu deployments
```

Reply:
@tallenaz this seems to have disappeared from the most recent push.  Maybe you force pushed over it?

Reply:
Thanks for spotting that. I just pushed up a commit for it.